src/pages/patterns/read-only-states-pattern/index.mdx

```diff
@@ -0,0 +1,365 @@
+---
+title: Read-only states
+description:
+  Read-only states are applied to components that users can review but not
+  modify and remove a component’s interactive functions.
+---
+
+<PageDescription>
+
+Read-only states are applied to components when the user is allowed to review
+but not modify the component. It removed all interactive functions from the
```

Comment:
oops this is my own typo, should this be "removes" instead?
```suggestion
but not modify the component. It removes all interactive functions from the
```

Reply:
Oh, yep! Good catch.